editor/selectors.js

```diff
@@ -544,12 +544,8 @@ export function getBlockFocus( state, uid ) {
  * @param  {Object}  uid   Block unique ID
  * @return {Boolean}       Whether user is typing within block
  */
-export function isTypingInBlock( state, uid ) {
-	if ( ! isBlockSelected( state, uid ) ) {
-		return false;
-	}
-
-	return state.selectedBlock.typing;
+export function isTypingInEditor( state ) {
```

Comment:
Yeah they are being destructed. So either we rename it there, or change it to non destructing.